### PR TITLE
[chore] refactor to use new server endpoint

### PR DESCRIPTION
**Pull request by mmnoo**
- refactor new site and edit site form (for name and landscape info) to use new api and not mock landscape data
- refactor orgs list to use new api and not mock data
- added speculative ux for when there are no 'your orgs' or 'other orgs' in orgs list

Screen Shot 2022-06-19 at 10 10 48 AM
Screen Shot 2022-06-19 at 10 10 40 AM
Screen Shot 2022-06-19 at 10 10 08 AM
Screen Shot 2022-06-19 at 10 08 39 AM

### CODE REVIEW

**Review by julia-rom — COMMENTED** [review]
Just testing out the review apps link - if I click on the first site, then go to organizations, I get a 404 with a loading wheel that doesn't stop spinning. I don't see the 'You don't have any organizations' page that you posted in the screenshot @mmnoo

File: mrtt-ui/src/views/Organizations.js [hunk 1]
```diff
@@ -1,49 +1,64 @@
...
-    setOtherOrganizations(
-      mockOrganizations.filter((organization) => !organization.isCurrentUserMember)
-    )
+  const [isLoading, setIsLoading] = useState(true)
+  useEffect(function loadOrganizations() {
+    axios
+      .get(organizationsUrl)
+      .then(({ data }) => {
+        setIsLoading(false)
+        setYourOrganizations(data.filter((organization) => !!organization.isCurrentUserMember))
```
Comment (julia-rom) [inline mrtt-ui/src/views/Organizations.js:22]: the exclamation and double exclamation make this kind of hard to read/understand - is this the same as `organization?.isCurrentUserMember`

**Review by julia-rom — CHANGES_REQUESTED** [review]
Was expecting to see the empty orgs list from the screenshot in - `/organizations`, was instead of the 404/spinning wheel  - let me know if this is a bug or a todo!

**Comment by mmnoo** [comment]
> Was expecting to see the empty orgs list from the screenshot in - `/organizations`, was instead of the 404/spinning wheel - let me know if this is a bug or a todo!

def a bug.....

File: mrtt-ui/src/views/Organizations.js (hunk 1 above)
Comment (mmnoo) [reply mrtt-ui/src/views/Organizations.js:22]: ya, the !! probably isnt necessary here

**Pull request merged**